wagtail/admin/templates/wagtailadmin/pages/create.html

```diff
@@ -38,13 +38,15 @@ <h1 class="icon icon-doc-empty-inverse">
                         <div class="dropdown dropup dropdown-button match-width">
                             {% include "wagtailadmin/pages/_preview_button_on_create.html" with label=preview_label icon=1 %}
                             <div class="dropdown-toggle icon icon-arrow-up"></div>
-                            <ul role="menu">
-                                {% for mode_name, mode_display_name in preview_modes %}
-                                    <li>
-                                        {% include "wagtailadmin/pages/_preview_button_on_create.html" with mode=mode_name label=mode_display_name %}
-                                    </li>
-                                {% endfor %}
-                            </ul>
+                            <nav aria-label="actions">
```

Comment:
Hmm this one feels like it's not as good of a fit for a `nav` landmark, since it’s already within a `ul` that has other buttons, and it’s very close to the footer landmark. What do you think @helenb? Other option would be to have the `nav` around the `ul` that wraps all of this, I’m not sure which of the two would be the most pleasant to use.

Reply:
In this case I'll move the nav to outside the wrapping `ul`. I'm not convinced that `role="contentinfo"` makes sense on the `footer` nearby, and indeed Axe complains that this is nested in another landmark. But that is probably best addressed separately.